August.17 18

zsofi-gagyi opened the pull request.

zsofi-gagyi committed:
feat(end-to-end testing): set up cypress

zsofi-gagyi committed:
Merge branch 'master' into august.17-18

zsofi-gagyi committed:
test(end-to-end)

zsofi-gagyi committed:
test(end-to-end): more tests

zsofi-gagyi committed:
test(end-to-end): more tests

zsofi-gagyi committed:
refactor(integration tests)

zsofi-gagyi committed:
refactor(the order collaborators appear, guest data seeding)

zsofi-gagyi committed:
refactor(integration tests)

gagyibenedek commented on TaskManager/TaskManager/cypress/fixtures/example.json:
```diff
@@ -0,0 +1,5 @@
+{
```
Erre a filera biztosan szukseg van? :)

zsofi-gagyi committed:
refactor(integration tests): final corrections

zsofi-gagyi committed:
fix(unauthorized requests to API endpoints): return status code

instead of the old behaviour, where it redirected to the login page
(which is kept as correct behaviour for the non-API endpoints)
